src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -562,5 +562,14 @@ public static class Diagnostics {
 			defaultSeverity: DiagnosticSeverity.Error,
 			isEnabledByDefault: true
 		);
+
+		public static readonly DiagnosticDescriptor InconsistentMethodAttributeApplication = new DiagnosticDescriptor(
+			id: "D2L0076",
+			title: "Attribute is applied inconsistently to method definitions.",
+			messageFormat: "The [{0}] attribute is applied inconsistently between \"{1}\" and its source definition \"{2}\". This wiil cause inconsistent behaviour.",

```

Comment:
```suggestion
			messageFormat: "The [{0}] attribute is applied inconsistently between \"{1}\" and its source definition \"{2}\". This will cause inconsistent behaviour.",
```